org.eclipse.buildship.core.test/src/main/groovy/org/eclipse/buildship/core/internal/test/fixtures/GradleVersionConstraints.java

```diff
@@ -78,16 +78,6 @@ public boolean apply(GradleVersion element) {
                 }
             };
         }
-        if (trimmed.startsWith(NOT_EQUALS)) {
```

Comment:
remove this part since it will ignore https://github.com/eclipse/buildship/blob/117332457a6763c21a8ea7d602ed25b39ba8e92e/org.eclipse.buildship.core.test/src/main/groovy/org/eclipse/buildship/core/internal/test/fixtures/GradleVersionConstraints.java#L86-L94 which can also handle multiple `!=`.

Reply:
👍